db/objects/micronutrients/V0.1.0.50__micronutrients.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE micronutrients (
+	id                  text primary key
+	, name              text
+	, unit              text
+	, description       text
+	, category          text
```

Comment:
I'd move the code from line 8 to be on line 6, just to make it clear which column is involved; currently it looks (and possibly acts?) like a check constraint on fooditem_colname.

So: 

```suggestion
	, category          text CHECK (category IN ('vitamin', 'mineral', 'other'))
```

Reply:
Yep, that make sense - updated :).